### PR TITLE
MM-47605: Screen Reader not announcing Add members button

### PR DESCRIPTION
**Pull request by babinderrathi**
#### Summary
Focus is not going to the "Add" button in the "Channel Members" popup window, and the screen reader is not announcing the "Add" Members button.

#### Ticket Link

  Fixes https://github.com/mattermost/mattermost-server/issues/21419
  Fixes  https://mattermost.atlassian.net/browse/MM-47605

#### Related Pull Requests
None

#### Screenshots
https://user-images.githubusercontent.com/102028591/201334081-e4785486-266c-45d7-aed4-ff1dd4341fa7.mp4

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/channel_members_rhs/action_bar.tsx [hunk 1]
```diff
@@ -84,6 +84,18 @@ const ActionBar = ({className, channelType, membersCount, canManageMembers, edit
         };
     }, [handleShortcut]);
 
+    const intl = useIntl();
+
+    const manageAriaLabel = intl.formatMessage({
+        id: 'channel_members_rhs.action_bar.manage_button_aria',
+        defaultMessage: 'manage members ',
```
Comment (BenCookie95) [inline components/channel_members_rhs/action_bar.tsx:91]: We can remove the space at the end of the string

**Comment by BenCookie95** [comment]
> @BenCookie95 , Sir could you please take a look.

@babinderrathi The ticket has been updated https://mattermost.atlassian.net/browse/MM-47605.

Once the user clicks on the channel members button, the focus needs to switch the X in the RHS view and then traverse the RHS. You can see more details in the Jira ticket

**Comment by babinderrathi** [comment]
@BenCookie95 sir i applied some changes, can you please re-review.

**Comment by BenCookie95** [comment]
> @BenCookie95 sir i applied some changes, can you please re-review.

@babinderrathi Focus still isn't going to the RHS once you open the channel member list. Once the channel member list is opened, the focus should immediately change to the X on the RHS and if I close the RHS by pressing the X, focus should go back to the channel member button

**Comment by AshishDhama** [comment]
@babinderrathi Is something blocking you on this?

**Pull request closed without merging**